schema.py

```diff
@@ -1,5 +1,36 @@
 __version__ = '0.3.0'
 
+error_messages = {
+    'did not validate': '{schema} {reason} {data}',
```

Comment:
I think that I made this pull request over a year ago and it's obvious that this project is not actively maintained.  I've moved on.  [schemtatics](https://github.com/schematics/schematics) looks interesting....